### PR TITLE
Add mimalloc 2.1.7-bcr.alpha.1

### PR DESCRIPTION
**Pull request by fmeum**

### CODE REVIEW

**Comment by bazel-io** [comment]
Hello @bazelbuild/bcr-maintainers, modules without existing maintainers (mimalloc) have been updated in this PR. Please review the changes.

**Comment by fmeum** [comment]
@meteorcloudy I sent https://github.com/bazelbuild/continuous-integration/pull/2004 to fix the CI failure.

File: modules/mimalloc/2.1.7-bcr.alpha.1/source.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "integrity": "sha256-Du05MZ8Tmv3oUVAQ/1m68k3p5H6jFqMVOY6AJ9GYIC0=",
+    "strip_prefix": "",
```
Comment (meteorcloudy) [inline modules/mimalloc/2.1.7-bcr.alpha.1/source.json:3]: Should be `mimalloc-2.1.7`?

**Comment by fmeum** [comment]
@keith @illicitonion Could you review this if you have time? I want this to be as easy to use as possible while still (eventually) exposing the relevant configuration options.

**Review by illicitonion — APPROVED** [review]
LGTM! Thanks!

**Review by keith — APPROVED** [review]
nice, i actually just last week wanted this

**Pull request merged**